windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
```

Comment:
From my point of view it's probably not that important; I'd just leave a comment so that someone coming along later doesn't assume that we use Ubuntu for everything.